app/views/chargeback/_cb_rate_edit_table.html.haml

```diff
@@ -18,13 +18,14 @@
       %th= _("Fixed")
       %th= _("Variable")
   %tbody
-    - @edit[:new][:details].sort_by { |rd| [rd[:group], rd[:description], rd[:sub_metric].to_s] }.each do |detail|
+    - @edit[:new][:details].sort_by { |rd| [rd[:group], rd[:description], rd[:sub_metric].to_s] }.each_with_index do |detail, detail_index|
```

Comment:
@hstastna If I'm not wrong, creating `detail_index` here has no effect, as it's changed few lines deeper.